src/services/shims.ts

```diff
@@ -951,10 +951,10 @@ namespace ts {
             );
         }
 
-        public getOutliningSpans(fileName: string): string {
+        public getOutliningSpans(fileName: string, fileSize: number): string {

```

Comment:
Do you need to add the fileSize to the call? Doesn't the document registry already have the file as a string if we've already parsed it (making string.length available)?

Reply:
(see IScriptSnapshot.getLength() )